docs/adr/0001-retiring-licence-finder.md

```diff
@@ -0,0 +1,82 @@
+
+### Problems with the current approach
+
+Only a few licences can be considered "mainstream". Most are for a specialist activity. However as licences are managed in Mainstream Publisher only GDS content designers can create licence pages.  As Content Designers across government cannot create licence pages, they have taken to using other Whitehall formats to create licence content. This means that the licences created in other formats do not have a licence identifier, and therefore do not appear in the licence-finder results.
+
+Another reason that not all licences appear in the licence finder results is because a developer is needed to updated the CSV files and run a rake task, a step that is easy to miss.
+
+It is possible for no results to be displayed to the user if they do a text search. For example, "restaurant" doesn't yield any results, even though it is in the name of a sector.
+
+Design-wise, the existing tool doesn't match any of the other design patterns on GOV.UK (e.g. smart-answers or finders), nor does it use any design system components.
+
+Finally, the existing tool doesn't contain any extra analytics tracking on the search field or the [sectors] and [activities] list making it difficult to track user behaviour.
+
+## Decision
+
+This Licence Finder application will be retired and replaced with a specialist finder similar to the [Air Accidents Investigation Branch reports] finder.
+
+Specialist Finders find documents with a particular special document schema and are configured in [Specialist Publisher].
+Specialist Finders are an existing pattern on GOV.UK and follow the design system. The specialist finder render already contains analytics tracking for the facets, search box and results clicked.
+
+Another benefit of this approach is that licences will still appear in the GOV.UK site search. Also, the default behaviour of finders is to show all of the available licences to the user when the finder loads, so the user will always see some results.
+
+A new licence document type will be created in Specialist Publisher. This new document type in Specialist Publisher will allow content designers to populate all of the existing fields for licence, as well as tagging the individual licence to sectors and activities via a user interface rather than a rake task. Specialist Publisher already allows publishers in other departments to create content by either being part of a named organisation, or by being assigned a custom permission. This will move the burden of managing licence content to the department that owns them.
+
+Specialist Publisher also generates tagging interfaces for specialist documents from the configuration files, so a new interface will not need to be built.
+
+### Other approaches considered
+
+#### Updating the current tool
+
+This approach was immediately discarded for two main reasons. The current tool is not fit for purpose, and the overhead of needing to update so many licences at once.
```

Comment:
I'd remove "immediately" from that sentence. We've spent ages saying we're not sure if we're going to iterate on it before figuring out what to do instead.

Reply:
Could link back up to the 'Problems with the current approach', as this section outlines these issues